src-v2/Judge/NTS.Judge/Adapters/Behinds/DashboardBehind.cs

```diff
@@ -36,7 +44,20 @@ async Task SafeStart()
             throw new DomainException("Cannot start - event is not configured");
         }
         await CreateEvent(setupEvent);
+        if (!setupEvent.Officials.Any())
+        {
+            throw new DomainException("Cannot start - officials aren't configured");
+        }
         await CreateOfficials(setupEvent.Officials);
+        if (!setupEvent.Competitions.ElementAt(0).Phases.Any())
+        {
+            throw new DomainException("Cannot start - phases aren't configured");
+        }
+        if (!setupEvent.Competitions.ElementAt(0).Contestants.Any())
+        {
+            throw new DomainException("Cannot start - contestants aren't configured");
+        }
```

Comment:
This isn't a very good validation:
1. Competitions may be empty, which will throw unhandled exception
2. You are only validating the first competition, which means you can start with second competition without phases
3. The message isn't very descriptive, because the user may have configured phases, but omitted one of the competitions.